src/components/cardbuilder/card.css

```diff
@@ -81,7 +80,6 @@ button {
     transition: none;
     border: 0 solid transparent;
     /* These both are needed in case cardBox is a button */
```

Comment:
I would also remove any comments related to the vendor prefixes like this one.

Reply:
Done.

There's still a few prefixed stuff left, like animations, but I think we need to do more cleanup and refactoring before removing these safely.

Reply:
Oops, I hadn't read you're entire comment about the leftover prefixes. We can leave them for now, but I definitely want to remove the rest of the related comments since they won't make sense without the original context.

Reply:
Oops, I hadn't read your entire comment about the leftover prefixes. We can leave them for now, but I definitely want to remove the rest of the related comments since they won't make sense without the original context.

Reply:
I ended up going over the keyframes and some background gradients to remove the vendor-prefixed parts, since autoprefixer takes care of that.

I also cleaned up the few @supports statements I found.